core/src/main/java/com/datastax/oss/driver/internal/core/loadbalancing/DefaultLoadBalancingPolicy.java

```diff
@@ -252,7 +253,9 @@ public void onNodeError(
       @NonNull DriverExecutionProfile executionProfile,
       @NonNull Node node,
       @NonNull String logPrefix) {
-    updateResponseTimes(node);
+    if (!(error instanceof DriverTimeoutException)) {
```

Comment:
@SiyaoIsHiding are you sure this can happen? My recollection is that when a driver timeout happens, the query is aborted and `RequestTracker.onError()` is invoked, not `RequestTracker.onNodeError()`.

https://github.com/apache/cassandra-java-driver/blob/16260261d3df50fcf24fac1fc2d37896c4a111bf/core/src/main/java/com/datastax/oss/driver/internal/core/cql/CqlRequestHandler.java#L209-L211